org.nxengine.nxengine_evo.json

```diff
@@ -0,0 +1,49 @@
+{
+	"app-id": "org.nxengine.nxengine_evo",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "21.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "nxengine-evo",
+	"finish-args": [
+		"--socket=fallback-x11",
+		"--share=ipc",
+		"--socket=wayland",
```

Comment:
does it really support wayland?

Reply:
Yes it does support wayland because it uses SDL2, which supports wayland.

Tested on a Steam Deck in game mode.